test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/DoublePredicateTest.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import java.util.function.DoublePredicate
+
+import org.junit.Assert._
+import org.junit.Test
+
+import org.scalajs.testsuite.utils.AssertThrows._
+
+class DoublePredicateTest {
+  import DoublePredicateTest._
+
+  private val largerThan10 = makePredicate(_ > 10.0D)
+  private val even = makePredicate(_ % 2 == 0.0D)
```

Comment:
```suggestion
  private val largerThan10 = makePredicate(_ > 10.0d)
  private val even = makePredicate(_ % 2 == 0.0d)
```
or
```suggestion
  private val largerThan10 = makePredicate(_ > 10.0)
  private val even = makePredicate(_ % 2 == 0.0)
```
The same applies everywhere in this file.